application/routes/term.py

```
@@ -35,30 +37,34 @@
 def start_terminal():
     session_id = request.json.get('sessionID')
     load_check = request.json.get('loadCheck', True)
-
+    logger.debug("Term: Starting terminal session: %s", session_id)
     def generate():
```

Ruff (0.8.2)

Missing return type annotation for private function `generate`

(ANN202)